tests/sentry/workflow_engine/processors/test_detector.py

```diff
@@ -29,9 +29,7 @@ class NoHandlerGroupType(GroupType):
 
         class MockDetectorHandler(DetectorHandler[dict]):
             def evaluate(self, data_packet: DataPacket[dict]) -> list[DetectorEvaluationResult]:
```

Comment:
Does this still need to take `data_packet`?

Reply:
Yeah, evaluate always accepts only a DataPacket